src/lib/integration_test_cloud_engine/mina_automation.ml

```diff
@@ -173,7 +173,7 @@ module Network_config = struct
       | [] ->
           mp
       | hd :: tl ->
-          let ( { Test_config.Test_Account.balance; account_name; timing }
+          let ( { Test_config.Test_Account.balance; account_name; timing; _ }
               , (pk, sk) ) =
```

Comment:
The `_` after `timing` is what I had to add for the code to compile when I reintroduced the key-pair generation for gcp.

Reply:
Now it's showing. That's fine and needed because I added `pk` and `sk` fields directly to `Test_account.t`. All testing engines use the `Test_config` module, but the GCP engine just doesn't need these fields in `Test_account.t` because they don't pre-generate the keys